components/list/README.md

```diff
@@ -92,3 +92,10 @@ The `d2l-list-content` provides additional consistent layout for primary and sec
   </d2l-list-item-content>
 </d2l-list-item>
 ```
+
+## Future Enhancements
+
+- Drag & drop support
+- Paging: integration with "load more", "scroll" and "numeric" paging mechanisms
```

Comment:
@m6jones @dbatiste anything else you're aware of that we'd like to add?

Reply:
I think Maya is keen to update the list items to use grid for managing focus with arrow keys.